ixmp/core.py

```diff
@@ -465,6 +481,13 @@ def add_timeseries(self, df, meta=False):
             - `year` and `value`—long, or 'tabular', format.
             - one or more specific years—wide, or 'IAMC' format.
 
+            To support subannual temporal resolution of timeseries data, a
+            column `subannual` is optional in `df`. The entries in this column
```

Comment:
It may be a confusion as somewhere it called `time`, not `subannual`

Reply:
where is it called `time` in the public-facing part?

Reply:
I guess as dataframe column name. Just wanted to make an accent that we may want to rename it (e.g. in parent PR). See test_new_subannual_timeseries_as_iamc test

Reply:
For me, it's important that all public-facing interfaces (both getter and setter) in `ixmp` and the RestAPI use the column name `subannual` and `timeslices` for the allowed values (unless someone comes up with an alternative in the next 48 hours). Whether it makes sense to change that throughout the ixmp Java backend and the database schemes - I leave that decision to you.

The test `test_new_subannual_timeseries_as_iamc` is as I expect it (well, preferably lower-case `summer`).